mkosi/backend.py

```diff
@@ -951,3 +957,22 @@ def complete_step(cls, text: str, text2: Optional[str] = None) -> Iterator[List[
 
         if text2 is not None:
             cls.print_step(text2.format(*args))
+
+
+def chown_to_running_user(paths: Union[PathString, [PathString]]):
```

Comment:
Our python version is still locked to 3.7, so this should use `Sequence[PathString]` instead of `[PathString]`